tests/platform_tests/test_reboot.py

```diff
@@ -236,11 +236,10 @@ def test_watchdog_reboot(duthosts, enum_rand_one_per_hwsku_hostname, localhost,
     @summary: This test case is to perform reboot via watchdog and check platform status
     """
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
-    py_res = duthost.command("python -c \"import sonic_platform\"", module_ignore_errors=True)
-    test_watchdog_supported = "python{} -c \"import sonic_platform.platform as P; P.Platform().get_chassis().get_watchdog(); exit()\"".format('3' if py_res['failed'] else '2')
+    test_watchdog_supported = "watchdogutil status"
```

Comment:
Variable isn't really necessary to hold this command. Suggest removing this line and changing lines 241-242 to

```python
watchdogutil_status_result = duthost.command("watchdogutil status", module_ignore_errors=True)
if "" != watchdogutil_status_result["stderr"] or "" == watchdogutil_status_result["stdout"]:
```